File: bin/com/problem/go/googleKickstart/2022/roundg/walktober.go
package main

import "fmt"

func main() {
  var T int
  fmt.Scan(&T)

  for t := 0; t < T; t++ {
    var n, m, p, ans int
    fmt.Scan(&n, &m, &p)

    fmt.Println(n, m, p, ans)

    scoreBoard := make(map[int][]map[int]int)

    for i := 1; i <= n; i++ {
      var score [3]int
      for j := 1; j <= m; j++ {
        var s int
        fmt.Scan(&s)
        fmt.Println(s)
        score[j - 1] = s

      }
      fmt.Println(score)
    }

    fmt.Println(scoreBoard)


  }
}

File: bin/com/problem/cpp/googleKickStart/roundg/walktober.cpp
#include <bits/stdc++.h>
#include <vector>

using namespace std;

int main() {
  int T;
  cin >> T;

  for (int t = 1; t <= T; t++) {
    int N, M, P, ans = 0;
    cin >> N;
    cin >> M;
    cin >> P;

    vector<vector<int>> scoreBoard;
    for (int i = 0; i <= N; i++) {
      vector<int> score;
      for (int j = 0; i <= M; i++) {
        int S;
        cin >> S;
        score.push_back(S);
      }
      scoreBoard.push_back(score);
    }

    for (int i = 0; i <= scoreBoard.size(); i++) {
      cout << scoreBoard[i][0] << endl;
    }
    /*
    cout << score.size() << endl;
    for (int i = 0; i <= score.size(); i++) {
      cout << score[i] << endl;
    }
    */
  }
}
